playground/frontend/playground_components/lib/src/models/example.dart

```diff
@@ -35,6 +35,7 @@ class Example extends ExampleBase {
     this.graph,
     this.logs,
     this.outputs,
+    super.datasets = const [],
```

Comment:
1. Sort among super.
2. Why is it required for the super and not required here?

Reply:
required now, done